Bottom: 0adb2e7a2b09a6b308abfe5fbf618d5b92328f22
Top:    713f811d976e2eafb351d5977cbff940881f460d
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 20:31:12 +0000

Re-enable soldermask emission


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index efe337c..8d81a8e 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1040,7 +1040,7 @@ object3d_from_soldermask_within_area (POLYAREA *area, int side)
   LayerType *layer;
   POLYAREA *pa;
 
-  return NULL;
+//  return NULL;
 
   poly_Copy0 (&info.poly, area);
   info.side = side;
